action_server/src/sema4ai/action_server/_cli_impl.py

```diff
@@ -905,6 +906,27 @@ def _use_db_ctx(*args, **kwags):
                     if code != 0:
                         return code
 
+                # Ok, we're starting the server. At this point, if there are any
+                # runs in the `not_run` state, or in the `running` state, we should
+                # mark them as cancelled.
+
+                runs_to_cancel = db.all(
```

Comment:
it feels like having some reusable function for cancelling all pending runs could come in handy ^^ minor ofc

Reply:
I guess that if some other place ends up needing this can be refactored (but I'd rather leave as is for now).